cmd/new.go

```diff
@@ -166,12 +166,8 @@ func createAndEditSnippet(newSnippet snippet.SnippetInfo, snippets snippet.Snipp
 }
 
 func countSnippetLines() int {
-	filepath, err := config.ExpandPath(config.Conf.General.SnippetFile)
-	if err != nil {
-		return 0
-	}
-
-	f, err := os.Open(filepath)
+	// Count lines in snippet file
+	f, err := os.Open(config.Conf.General.SnippetFile)
```

Comment:
How come we don't need the expand here anymore?

Reply:
I think we need to find a way to document the expand usage/lack of usage - it's tricky to understand. 

I feel I've asked this same question or the same question reversed quite a few times, and I'm someone familiar with this PR and codebase 😄 

A comment is obviously easiest, but wonder if there's a better way even.

Reply:
Yeah, we've circled back to this a couple of times. it definitely warrants some better way to document it. 
I'll see what I can come up with.

Reply:
hate to admit it, but it may have been a bad rebase job on my end. 
Expanding path when reading/accessing file is what we want.

Reply:
Yeah I thought so, I'm reviewing this thoroughly though don't worry 🙂

Reply:
so, I tried wrapping those os calls, but they pose a problem, which I'm not particularly fond of: our wrapper functions would hide away the original documentation (think mouse hover over function).

I made sure we add Expand() directly where accessing a file / a dir is required.

Reply:
My only concern with our current approach is this:

Expansion is always necessary when we use the SnippetFile property in our config. Always.
So right now we are leaving room for devs to forget to call Expand.

It would be really nice if we find a way to automatically expand this filename wherever it is accessed. Think "getter method" on `config.Conf.General.SnippetFile`. Maybe an interface method on GeneralConfig - GetSnippetFilePath(). What do you think of that approach? Any other ideas to deal with this?

Reply:
There are two use cases for SnippetFile: full path when a file is to be accessed and relative path in any other cases. Problem with a single getter is that it is still tricky to indicate which version to return. 

We can do two getters, or simpler with an additional field called `SnippetFileAbsPath`. This field is hidden from user, auto populated from `SnippetFile` value when config is loaded and it is used specifically for accessing file (documentation could help)

thoughts?